src/test/resources/featureFiles/loginTest.feature

```diff
@@ -0,0 +1,49 @@
+Feature: Login Page Test
+  Background:
+    Given user is on the login page "https://www.saucedemo.com/"
+  Scenario Outline: Valid username and password
+    When user enters valid username and password
```

Comment:
In acest scenariu nu-ti va lua userName / password , nu-s puse corect in Scenario Outline

Reply:
Multumesc! Am modificat.